### ISSUE TITLE
Unable to install in mac

### ISSUE THREAD

**Issue by sstha** [CAIDA/libipmeta#41]
I am trying to install pyipmeta in mac but unable to install with the error

% python setup.py build_ext
running build_ext
building '_pyipmeta' extension
creating build/temp.macosx-11.5-x86_64-2.7
creating build/temp.macosx-11.5-x86_64-2.7/src
cc -fno-strict-aliasing -fno-common -dynamic -g -Os -pipe -fno-common -fno-strict-aliasing -fwrapv -DENABLE_DTRACE -DMACOSX -DNDEBUG -Wall -Wstrict-prototypes -Wshorten-64-to-32 -iwithsysroot /usr/local/libressl/include -DNDEBUG -g -fwrapv -Os -Wall -Wstrict-prototypes -DENABLE_DTRACE -pipe -I/System/Library/Frameworks/Python.framework/Versions/2.7/include/python2.7 -c src/_pyipmeta_module.c -o build/temp.macosx-11.5-x86_64-2.7/src/_pyipmeta_module.o
In file included from src/_pyipmeta_module.c:35:
src/_pyipmeta_provider.h:36:10: fatal error: 'libipmeta.h' file not found
#include <libipmeta.h>
         ^~~~~~~~~~~~~
1 error generated.
error: command 'cc' failed with exit status 1

libipmeta (>= 3.1.0) was missing so tried installing libipmeta with Install via clone method but installation failed

%git clone https://github.com/CAIDA/libipmeta.git
Cloning into 'libipmeta'...
remote: Enumerating objects: 1979, done.
remote: Counting objects: 100% (193/193), done.
remote: Compressing objects: 100% (85/85), done.
remote: Total 1979 (delta 131), reused 153 (delta 108), pack-reused 1786
Receiving objects: 100% (1979/1979), 577.12 KiB | 892.00 KiB/s, done.
Resolving deltas: 100% (1410/1410), done.
% cd libipmeta
 % git submodule init
Submodule 'common' (https://github.com/CAIDA/cc-common.git) registered for path 'common'
% git submodule update
Cloning into '/libipmeta/common'...
Submodule path 'common': checked out '68eff51d81143604e311ad0b37e22cbe7e591c7a'
./autogen.sh
Initializing submodules
Running autoconf
autoreconf: export WARNINGS=
autoreconf: Entering directory '.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -I m4
aclocal: warning: couldn't open directory 'm4': No such file or directory
configure.ac:47: error: AM_INIT_AUTOMAKE expanded multiple times
/usr/local/Cellar/automake/1.16.5/share/aclocal-1.16/init.m4:29: AM_INIT_AUTOMAKE is expanded from...
configure.ac:38: the top level
/usr/local/Cellar/automake/1.16.5/share/aclocal-1.16/init.m4:29: AM_INIT_AUTOMAKE is expanded from...
configure.ac:47: the top level
autom4te: error: /usr/local/opt/m4/bin/m4 failed with exit status: 1
aclocal: error: /usr/local/Cellar/autoconf/2.71/bin/autom4te failed with exit status: 1
autoreconf: error: aclocal failed with exit status: 1

**Comment by alistairking** [comment]
Yeah, I think I came across this myself with the latest version of autoconf/automake available via brew. I ended up just using docker, but a PR to fix the build scripts would be much appreciated :)
